images.yaml

```diff
@@ -96,29 +92,22 @@
     tag: "1.0"
   - sha: a64c8ed5df00c9f238ecdeb28eb4ed226faace573695e290a99d92d503593e87
     tag: "1.2"
-- name: gcr.io/google_containers/kube-state-metrics
-  tags:
-  - sha: 953a3b6bf0046333c656fcfa2fc3a08f4055dc3fbd5b1dcdcdf865a2534db526
-    tag: v1.2.0
-- name: gcr.io/google_containers/nginx-ingress-controller
-  tags:
-  - sha: 03fd8fc46018d09b4050d4daaf50bff73c80936994b374319ed33cbb2c1684f4
-    tag: 0.9.0-beta.11
 - name: gcr.io/google_containers/pause-amd64
   tags:
   - sha: 59eec8837a4d942cc19a52b8c09ea75121acc38114a2c68b98983ce9356b8610
     tag: "3.1"
 - name: gcr.io/heptio-images/eventrouter
-  tags:
-  - sha: dba60a88600d2d94fcd4c365e2931e54ae9aa495e94a924f80814e019b7ea046
-    tag: v0.3
+  patterns:
+  - pattern: '>= v0.3'
```

Comment:
Why a pattern here? Are we updating these automatically?

Reply:
We are not updating them, so I can leave the tag. I was thinking it was better to have all versions since we not paying for storage and images anyway but maybe we can keep the tags for those. I just think we should have all versions so we do not wait on retagger PR as this could become a blocker.
@stone-z do you prefer me to switch to tags for those?

Reply:
Pattern jobs make every retagger run longer and potentially more prone to failing on aliyun. Yes we don't pay for storage currently, but we pay for CircleCI, and what's the point of collecting thousands of images we'll never deploy? As I said, not a blocker, just a preference of mine that we continue to use tags for slow-changing images without automation already in place